Assets/Python/pyWB/CvWBDesc.py

```diff
@@ -1809,7 +1809,11 @@ def write(self, f, sign):
 		f.write("BeginSign\n")
 		f.write("\tplotX=%d\n" %(sign.getPlot().getX(),))
 		f.write("\tplotY=%d\n" %(sign.getPlot().getY(),))
-		f.write("\tplayerType=%d, (%s)\n" %(sign.getPlayerType(), gc.getPlayer(sign.getPlayerType()).getName().encode(fileencoding)))
+		iPlayer = sign.getPlayerType()
```

Comment:
This looks weird on Github, as if there was an additional linebreak. Are there trailing whitespaces?

Reply:
The is no additional linebreak. There are also no trailing whitespaces. I guess it is just Github being weird.